pkg/kubeapiserver/admission/initializer.go

```diff
@@ -67,6 +75,14 @@ func (i pluginInitializer) Initialize(plugin admission.Interface) {
 		wants.SetInternalClientSet(i.internalClient)
 	}
 
+	if wants, ok := plugin.(WantsToRun); ok {
```

Comment:
@derekwaynecarr @deads2k any consensus on that? I can extend the admission controller with the Run method and call it at every place where it is used. Is this what is suggested in the comments? We can do it now or later, once the openshift use case is more clear? I don't have any strong preference over any of them.